addons/mail/static/src/models/channel_command/channel_command.js

```diff
@@ -10,7 +10,8 @@ function factory(dependencies) {
 
     ChannelCommand.fields = {
         /**
-         * FIXME use this value task-2343850
+         * Determines on which channel types `this` is available.
+         * Type of the channel (e.g. 'chat', 'channel' or 'groups')
```

Comment:
Current comment is good, but it would be best if it also specified the field should contain an array when filtering is desired, and should contain undefined when all types are allowed.

Reply:
Done